docs/src/modules/components/overview/tree-view/mainDemo/figma/FigmaExample.tsx

```diff
@@ -1,11 +1,16 @@
 import * as React from 'react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTheme } from '@mui/material/styles';
 import Stack from '@mui/material/Stack';
 import FigmaTreeView from './FigmaTreeView';
 import FigmaCard from './FigmaCard';
 import { IdType } from './items';
 
 export default function FigmaExample() {
   const [selectedItem, setSelectedItem] = React.useState<IdType | null>(null);
+  const docsTheme = useTheme();
+  const isMd = useMediaQuery(docsTheme.breakpoints.up('md'));
```

Comment:
You risk a flickering on the hydration with this approach.
Maybe you could

1. Add `defaultMatches: true` to make sure it always render on server
2. Add a CSS selector on the `Stack` to hide it on `md` and below 

That way, I think we shouldn't ever have flickering, and we don't render it on client if we doesn't need it.
Or we always render it and just hide it.